src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int FIELDS = 2;
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
too big method, let's split it into three small ones (readFromFile, createStatistic, writeToFile)